README.rst

```diff
@@ -12,8 +12,7 @@ embedded content (such as photos or videos) from third parties into your Plone
 site.
 
 You can easily embed content from Flickr, SlideShare, SoundCloud, Vimeo,
-Wikipedia, YouTube, and many other supporting the `oEmbed`_ format
```

Comment:
@jeanferri what was removed was the dependency on collective.oembed; the package still grabs the information using oEmbed, so this paragraph should be left without changes.